### ISSUE TITLE
you are in clear violation of mad's gpl license with mp3 releases

### ISSUE THREAD

**Issue by MinaciousGrace** [TeamRizu/OutFox#32]
seems like licenses only matter for forks that aren't this one

https://i.imgur.com/zMvoiFX.png
I don't believe this for two seconds, first of all :

"The MAD library (included with most Windows binary distributions), and
interface code in RageSoundReader_MP3, is under the GPL, available in
Copying.MAD.  Support for this library can be disabled at compile-time."

is still present in your own license file. So I would have to believe that after throwing the enormous shitfit you guys did when our fork had a singular closed component of code running under 1000 lines violating MAD's gpl license, it just happened to slip your mind that it is an issue that you should be dealing with in a good faith way. As if updating the license file due to the mp3 support being the only gpl licensed code, wouldn't be the primary purpose behind replacing the library, and wouldn't be the first thing you did. Doesn't pass the laugh test. Secondly there is no mention post 2018 ("research alternatives") in your dev history relating to mp3s, and looking at the symbols inside the executable_ZN19RageSoundReader_MP319do_mad_frame_decodeEb and _ZN19RageSoundReader_MP313MADLIB_rewindEv still exist as of this current release.

Conveniently for you your fork's entire codebase is _still_ closed after years, and seemingly all your shills who waxed poetic about how 0.01% of a codebase being closed is "destroying development" aren't going to say anything about the blatant hypocrisy. I know you're going to lie about replacing the library. I know you're not going to provide any proof. I know you're going to close this issue without responding. 

I'm just calling you out for it.

**Comment by Scraticus** [comment]
We have removed libmad from the repo, and replaced it with one that has a more alpha phase friendly licence. We have been trying to update and fix dependencies on things that are obviously not for a closed source alpha project.

We have moved to a header only mp3 driver called dr_mp3
![dr_mp3](https://user-images.githubusercontent.com/11047768/87860488-b841ef80-c935-11ea-8e00-efd0fa895889.png)

As for the 'close without responding', I am aware we did not have the best interactions with each other in the beginnings of this project, but you comparing us to older SM-Devs who caused a lot of the negativity which started the split continually and being somewhat 'trolling' to us also didn't help matters. However, we should not have responded as petty as we did, and for that I apologise Mina. I do not want there to be any more animosity between our projects and forks, whether they are for SM or otherwise. I hope you see this reply as moving past this assumption we still act and behave as we did in the middle of 2018.

I will show the top half of our 'RageSound_ReaderMP3.cpp' just to show we are no longer using libmad at this time.

![ragesoundreadermp3](https://user-images.githubusercontent.com/11047768/87860553-6057b880-c936-11ea-997d-dd167d156dd3.png)

I have also posted a list of the dependencies on Visual Studio, which no longer has libmad within them.
![dependencies](https://user-images.githubusercontent.com/11047768/87860618-b62c6080-c936-11ea-9de6-8c5ac1f80ecd.png)

I hope you take on board what I said, our games have much to benefit with us spending time working positively on bettering the experience for everyone, rather than infighting and sniping all the time which is frankly quite a waste of our resources and skill.

I thank you for reaching out to us, and I hope this satisfies your curiosity on our current MP3 Driver.

Be well.
